client/components/post-job/KeywordSelect.tsx

```diff
@@ -0,0 +1,18 @@
+import {skills} from '../../constants/skills';
+
+import {Box, Heading, Container, Flex, Text, Link, Image, SimpleGrid, HStack, Tag, TagLabel} from "@chakra-ui/react"
```

Comment:
Bunch of unused imports, no?

Reply:
Haven't finished this component yet. My bad for pushing this. It's not accessible anyway. Should finish it today